AdrianSimionov/obsidian-mindmap-nextgen
# Implement  checkmarks

**Issue #34:** [link](https://github.com/AdrianSimionov/obsidian-mindmap-nextgen/issues/23)

### What has been done?

Implementation of copying to clipboard with high quality output.

### How to test it?

1. Run the plugin
2. Open obsidian or restart plugin
3. Open a markdown document
4. Type any `- [ ]` option
5. Verify the result on rendered markmap
6. If there's error or should be rendered different, ask in the comments.
